src/clj/ataru/tarjonta_service/tarjonta_client.clj

```diff
@@ -0,0 +1,20 @@
+(ns ataru.tarjonta-service.tarjonta-client
+  (:require
+    [cheshire.core :as json]
+    [oph.soresu.common.config :refer [config]]
+    [org.httpkit.client :as http]
+    [taoensso.timbre :refer [warn]]))
+
+(defn get-hakukohde [hakukohde-oid]
+  (let [url      (str (get-in config [:tarjonta-service :hakukohde-base-url]) hakukohde-oid)
+        response @(http/get url)
+        status   (:status response)
+        result   (when (= 200 status)
+                   (-> response :body (json/parse-string true) :result))]
+    (if result
```

Comment:
404 will be returned. I don't think is a case that should normally occur, unless a form is deleted while still being in use?